### ISSUE TITLE
`onClick` events within map on mobile don't work

### ISSUE THREAD

**Issue by wallzero** [transcriptic/react-map-interaction#43]
This is somewhat a continuation of #31.

Before the previous fix, all touch events on the page weren't firing. Now, `onClick` events within the `<MapInteraction>` component fail to fire.

In the following example, `onClick` works fine in desktop mode but fails mobile. To test, try clicking on the nodes: [ComplexDiagram](https://wallzero.gitlab.io/react-ag/#/en/ComplexDiagram)

As a work around, I can use a combination of `onMouseUp` and `onTouchEnd`.

**Comment by scottyantipa** [comment]
@wallzero With the recent fixes, is this still an issue? 

See the section in the readme about handling click events:
```
### Click and drag handlers on child elements
This component lets you decide how to respond to click/drag events on the children that you render inside of the map. To know if an element was clicked or dragged, you can attach onClick or onTouchEnd events and then check the `e.defaultPrevneted` attribute. MapInteraction will set `defaultPrevented` to `true` if the touchend/mouseup event happened after a drag, and false if it was a click. See `index.stories.js` for an example.
```

The fix was in: #36

**Comment by wallzero** [comment]
Thanks it's working now!
